src/main/java/seedu/travelr/logic/commands/DeleteEventFromTripCommand.java

```diff
@@ -65,7 +66,8 @@ public CommandResult execute(Model model) throws CommandException {
         Event event = toDeleteFrom.getEvent(new Event(eventToDelete));
 
         toDeleteFrom.removeEvent(event);
-        model.addEvent(event);
+        model.returnToBucketList(event);
+        model.updateFilteredEventList(model.getBucketPredicate());
```

Comment:
Do we need to call this every time a command changes the bucket list? If I'm not wrong this line is to set the Events Display to show BucketList Events?

Reply:
Earlier on there was a bug where the display UI did not update when calling the add-et function, which was the reason why I added this method call to ensure that display UI updates correctly